_src/_assets/js/dashboard-charts.js

```diff
@@ -192,7 +205,7 @@
       width: chart.node().clientWidth * .9,
       yMax: d3.max(transformedData, function(d) {
         return d.value
-      }),
```

Comment:
are you using the prettier settings that are included with the project?
I autoformat before I commit code and it's sometimes reformatting your code
![image](https://user-images.githubusercontent.com/16123225/77800913-0bb1ca00-7035-11ea-9882-9027cb5abbd4.png)